compilers/concrete-compiler/compiler/lib/Runtime/simulation.cpp

```diff
@@ -183,6 +183,20 @@ void sim_wop_pbs_crt(
 
 uint64_t sim_neg_lwe_u64(uint64_t plaintext) { return ~plaintext + 1; }
 
+uint64_t sim_add_lwe_u64(uint64_t lhs, uint64_t rhs) {
+  if (lhs > UINT64_MAX - rhs) {
```

Comment:
should be UINT63_MAX (as well define UINT63_MAX) as we want to ensure that the padding bit is not hit